menpo/shape/mesh/textured.py

```diff
@@ -38,6 +38,15 @@ def __init__(self, points, tcoords, texture, trilist=None, copy=True):
         else:
             self.texture = texture.copy()
 
+    @property
+    def n_channels(self):
```

Comment:
Hmm, ok - it's not immediately clear to me that `n_channels` on a trimesh means the texture channels but I see the point about coloured meshes.